app/controllers/roles_controller.rb

```diff
@@ -0,0 +1,20 @@
+class RolesController < ApplicationController
+
+  def create
+    @role = Role.new(role_params)
+    if @role.save
+      redirect_to @role
```

Comment:
missing `notice: "Role is saved successfully."` ?

Reply:
notice added